hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/web/resources/TestParam.java

```diff
@@ -49,7 +51,7 @@
   @Test
   public void testAccessTimeParam() {
     final AccessTimeParam p = new AccessTimeParam(AccessTimeParam.DEFAULT);
-    Assert.assertEquals(-1L, p.getValue().longValue());
+    assertEquals(-1L, p.getValue().longValue());
```

Comment:
To avoid so much churn, let's just do the assertEquals in the new methods and leave this alone for the rest for now.
we can do a followup JIRA for that if so.

Reply:
Sure. Will revert the unrelated changes in this class.